### PR TITLE
Booking removes availability

### PR DESCRIPTION
**Pull request by alextwilson**
Doesn't fully resolve issue - booking not implemented

Added a button to each listing that deletes the space-availability relation
Added a delete /book route to perform this action
Moved listing a space with dates to spec helper

### CODE REVIEW

File: app/app.rb [hunk 1]
```diff
@@ -65,13 +71,16 @@ class DwellBNB < Sinatra::Base
 
   delete '/sessions' do
     session[:user_id] = nil
-    p current_user
     flash.keep[:notice] = 'Goodbye !'
     redirect to '/'
   end
 
+  delete '/book' do
```
Comment (Leigan0) [inline app/app.rb]: Please can this be changed its - I dont understand its purpose because it doesnt delete bookings so im 😕

**Review by ghost — COMMENTED** [review]
👍

**Comment by lewmoore** [comment]
:shipit:

**Comment by Leigan0** [comment]
👻

**Pull request merged**